Packs/ServiceNow/Playbooks/playbook-ServiceNow_-_Ticket_Management.yml

```diff
@@ -0,0 +1,843 @@
+    type: condition
+    task:
+      id: c45fcc66-2992-470f-8810-2ac6f6fa4640
+      version: -1
+      name: whether to open a ticket for this incident?
+      description: Checks whether to open a new ticket or add a comment to the task.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "11"
+      "yes":
+      - "16"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.Action
+            iscontext: true
+          right:
+            value:
+              simple: NewTicket
+          ignorecase: true
+      - - operator: isEmpty
+          left:
+            value:
+              simple: inputs.serviceNowTicketID
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -150,
+          "y": 760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 80014140-96c0-4da1-82a1-2c874cebfe90
+    type: regular
+    task:
+      id: 80014140-96c0-4da1-82a1-2c874cebfe90
+      version: -1
+      name: Update the ticket for the incident
+      description: Updates the specified ticket.
+      script: '|||servicenow-update-ticket'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      comments:
+        complex:
+          root: what is the comment for the ticket?.Answers
+          accessor: "0"
+      id:
+        complex:
+          root: inputs.serviceNowTicketID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1080,
+          "y": 1490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 3e5b6314-b03b-4fb0-82da-4a5990ef9b1f
+    type: title
+    task:
+      id: 3e5b6314-b03b-4fb0-82da-4a5990ef9b1f
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -150,
+          "y": 1690
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: de59d3d7-4a62-445e-8a1c-8a2bb052435b
+    type: condition
+    task:
+      id: de59d3d7-4a62-445e-8a1c-8a2bb052435b
+      version: -1
+      name: Is the comment defined on the input?
```

Comment:
I think "Was a comment provided in the input" ?

Reply:
Done.